### PR TITLE
Couple of bug fixes

### PR DESCRIPTION
**Pull request by pypingou**
- Make the fields readonly instead of disabling them (disable them mean they are not submitted so the form doesn't work :/)
- Ensure the review URL is a full URL not just the ticket identifier

### CODE REVIEW

File: pkgdb2/ui/packages.py [hunk 1]
```diff
@@ -1232,6 +1232,15 @@ def package_request_new():
         pkg_poc = flask.g.fas_user.username
         pkg_upstream_url = form.upstream_url.data
 
+        bz = APP.config.get('PKGDB2_BUGZILLA_URL')
+        if bz not in pkg_review_url:
+            try:
+                int(pkg_review_url)
+                pkg_review_url = APP.config[
+                    'PKGDB2_BUGZILLA_URL'] + '/' + pkg_review_url
```
Comment (puiterwijk) [inline pkgdb2/ui/packages.py]: You can reuse your "bz" variable here, not?

Comment (pypingou) [inline pkgdb2/ui/packages.py]: true

File: pkgdb2/ui/packages.py [hunk 2]
```diff
@@ -1232,6 +1232,15 @@ def package_request_new():
         pkg_poc = flask.g.fas_user.username
         pkg_upstream_url = form.upstream_url.data
 
+        bz = APP.config.get('PKGDB2_BUGZILLA_URL')
+        if bz not in pkg_review_url:
+            try:
+                int(pkg_review_url)
```
Comment (puiterwijk) [inline pkgdb2/ui/packages.py]: What would happen if bugzilla_url is https://bugzilla.redhat.com/, but I enter http://bugzilla.redhat.com/123456?
The URL would become http://.../123456/https://....
(or if I made a typo in the URL).

(just a note, I think people will just copy-paste normally)

**Comment by puiterwijk** [comment]
:+1:

File: pkgdb2/ui/packages.py (hunk 2 above)
Comment (pypingou) [inline pkgdb2/ui/packages.py]: It'll blow but won't be able to convert the url to an integer and thus will use the input as provided

**Pull request merged**